.github/workflows/android-build.yml

```
@@ -1,14 +1,6 @@
 name: Test Android build
 
-on:
-  pull_request:
-    paths:
-      - 'package/android/**'
-      - 'package/cpp/**'
-      - 'example/package.json'
-  push:
-    branches:
-      - main
+on: [push]
```

Nice one, should we also include the paths? It is possible to define under push as well:

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push

```suggestion
on: 
  push:
    paths:
       - 'package/android/**'
       - 'package/cpp/**'
       - 'example/package.json'
```